includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -40,6 +43,55 @@ public function track_product_updated( $product_id, $post ) {
 		WC_Tracks::record_event( 'product_edit', $properties );
 	}
 
+	/**
+	 * Track the Update button being clicked on the client side.
+	 * This is needed because `track_product_updated` (using the `edit_post`
+	 * hook) is called in response to a number of other triggers.
+	 *
+	 * @param WP_Post $post The post, not used.
+	 */
+	public function track_product_updated_client_side( $post ) {
+		wc_enqueue_js(
+			"
+			if ( $( 'h1.wp-heading-inline' ).text().trim() === 'Edit product') {
+				const initialStockValue = $( '#_stock' ).val();
+				let hasRecordedEvent = false;
```

Comment:
It looks like we need to stick to `var` here because we support IE 11 😞 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
😢 done